.changeset/mighty-regions-begin.md

```diff
@@ -0,0 +1,25 @@
+---
+"@biomejs/biome": minor
+---
+
+Added the rule [noExcessiveLinesPerFunction](https://biomejs.dev/linter/rules/no-excessive-lines-per-function/).
+This rule restrict a maximum number of lines of code in a function.
+
+The following code is now reported as invalid when the limit of maximum lines is set to 4:
+
+```js
+function foo() {
+  const x = 0;
+  const y = 1;
+  const z = 2;
+}
+```
```

Comment:
This looks like a function with 3 lines. Why is it invalid when max lines is set to 4?

Reply:
This example in the changeset still needs to be updated.

Reply:
Sorry, I understand. I'll update it.

Reply:
Done.